### DISCUSSION TITLE
New I18N feature

### DISCUSSION THREAD

**Discussion by laserir** [zauberzeug/nicegui#1526] [Q&A]
### Question

Hi, I've read about the new i18n feature in the changelog, but couldnt figure out if this is only for the (date...) quasar elements or also provides as way to switch languages in the UI?

**Comment by falkoschindler** [comment]
Sorry, I don't know what you mean. Can you link to the changelog you're referring to?

**Reply by laserir** [reply]
Yes, I was referring to this.

**Reply by falkoschindler** [reply]
Oh, I see. I didn't consider a feature from 4 months ago of being "new". 😄

**Comment by falkoschindler** [comment]
You are referring to the `language` parameter of the `ui.run` function (https://github.com/zauberzeug/nicegui/pull/874):

https://nicegui.io/documentation/run

This is, indeed, mainly for setting the language of Quasar elements like `ui.date`. The user code could access this setting and translate the UI accordingly, but there is no ready-to-use API for it, yet.
